README.md

```diff
@@ -218,19 +218,29 @@ When running OpenShift in a single VM, it is recommended to reuse the Docker dae
 for pure Docker use-cases as well.
 By using the same docker daemon as Minishift, you can speed up your local experiments.
 
-To be able to work with the docker daemon on your Mac or GNU/Linux host use the
-[`docker-env command`](https://minishift.io/docs/minishift_docker-env.html) in your shell:
+1. Make sure that you have the Docker client binary installed on your machine. For information about
+specific binary installations for your operating system, see the [Docker installation page](https://docs.docker.com/engine/installation/).
+
+1. Start Minishift with the `minishift start` command.
+
+1. Run the [`minishift docker-env`](https://minishift.io/docs/minishift_docker-env.html) command
```

Comment:
-1, this command does nothing. It does not export any variables. In fact you cannot export variables into the parent shell. You just need the `eval $(minishift docker-env)` call. That will evaluate the output of `minishift docker-env` in the context of my current shell.